farmdata2/farmdata2_modules/resources/BannerMessageConstants.js

```diff
@@ -16,6 +16,7 @@
  * </table>
  */
 var bannerMessageMap = {
+    standardError: {"msg": "Intermittent network error. Please try again later.", "class": "alert alert-danger alert-dismissible"},

```

Comment:
Forgot to add this to the documentation at the top of the page. I'll be sure to do that in my next commit.